### PR TITLE
Implement command for generating barcode 128

### PR DESCRIPTION
**Pull request by mknycha-ingrid**
[TA-460]

### CODE REVIEW

File: parsers/barcode_128.go [hunk 1]
```diff
@@ -22,10 +24,26 @@ func NewBarcode128Parser() *CommandParser {
...
 			if len(parts) > 1 {
 				if v, err := strconv.Atoi(parts[1]); err == nil {
+					// TODO: Add BY command and handle defaults
 					barcode.Height = v
 				}
 			}
 
+			if len(parts) > 2 && len(parts[2]) > 0 {
+				// The default here is 'Y', so we treat anything else than 'N' as true
+				barcode.Line = parts[2][0] != 'N'
```
Comment (fkryvyts-ingrid) [inline parsers/barcode_128.go:34]: it is probably ok to just use toBoolField here, right?
Even when Y is the default value
It just means that barcode.Line should be set to true when initialized

File: parser_test.go [hunk 2]
```diff
@@ -70,3 +70,49 @@ func TestDrawLabelAsPng(t *testing.T) {
 		})
 	}
 }
+
+func TestDrawLabelAsPng_Barcode128(t *testing.T) {
```
Comment (fkryvyts-ingrid) [inline parser_test.go:74]: Can be merged with the main test ( TestDrawLabelAsPng)

**Review by fkryvyts-ingrid — APPROVED** [review]

File: parsers/barcode_128.go (hunk 1 above)
Comment (mknycha-ingrid) [reply parsers/barcode_128.go:34]: It affects how the invalid value will work, but maybe we should not worry too much about the invalid domain.

File: parser_test.go (hunk 2 above)
Comment (mknycha-ingrid) [reply parser_test.go:74]: yes it can, merged

**Pull request merged**